src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -31,12 +31,18 @@ const AdContainer = styled.section`
   ${({ slotType }) => (slotType === 'mpu' ? mpuStyles : leaderboardStyles)}
 `;
 
-export const getBootsrapSrc = queryString => {
+export const getBootstrapSrc = queryString => {
   const useLiveSrc = isLive() || queryString.includes('ads-js-env=live');
   const params = useLiveSrc ? '' : 'test/';
   return `https://gn-web-assets.api.bbc.com/ngas/${params}dotcom-bootstrap.js`;
 };
 
+export const getBootstrapLegacySrc = queryString => {
+  const useLiveSrc = isLive() || queryString.includes('ads-js-env=live');
+  const params = useLiveSrc ? '' : 'test/';
+  return `https://gn-web-assets.api.bbc.com/ngas/${params}dotcom-bootstrap-legacy.js`;
```

Comment:
Do we want to move these URLs out into environment variables?

Reply:
Agree with this ^, otherwise LGTM

Reply:
We'd remove the ability to switch to the live one on the fly with the `ads-js-env=live` query string, unless we create 4 environment variables, 2 for the test script and 2 for the live scripts?

Reply:
`SIMORGH_ADS_SCRIPT_LIVE`
`SIMORGH_ADS_SCRIPT_LEGACY_LIVE`
`SIMORGH_ADS_SCRIPT_TEST`
`SIMORGH_ADS_SCRIPT_LEGACY_TEST`